tests/chainerx_tests/unit_tests/routines_tests/test_reduction.py

```diff
@@ -38,8 +38,18 @@
 ]
 
 
-@op_utils.op_test(['native:0', 'cuda:0'])
-@chainer.testing.parameterize_pytest('in_dtypes,out_dtype', [
+_cumsum_params = [
```

Comment:
Add test for `axis=None`.

Reply:
The current implementation doesn't support `axis=None` yet. In the native implementation, I use the `int` value of `axis` and have hence not defined `axis` to be of `OptionalAxes` type. Is there any way I could make it support `None` without modifying the type of `axis` argument?

Reply:
Please modify the type of `axis` argument.

Reply:
If I convert the type of the `axis` argument to `Axes`, how could I access the `int` value of it? I looked at the methods in https://github.com/chainer/chainer/blob/master/chainerx_cc/chainerx/axes.h but wasn't able to find any such way. Would you have any suggestions?

Reply:
Please use `absl::optional<int8_t>` as the type of `axis`.

Reply:
`CumsumKernel` is OK to take `int8_t axis`.

Reply:
With `axis` being `int8_t` type, I am not able to make use of the `GetSortedAxesOrAll` method that works with only `Axes` type. So, I get the default behavior(when `axis` argument is not provided) as `axis=0` instead of `axis=all`.

Reply:
`GetSortedAxesOraAll` seems not needed in this case because we don't need sorted axes.

Reply:
As temporary support, how about fixing to reshape the input array to 1-dim?

Reply:
I have made the changes. Tests pass for `axis=None` now.